### DISCUSSION TITLE
Undefined function tv:start/0. Inspect Mnesia ejabberd

### DISCUSSION THREAD

**Discussion by juancar0505** [processone/ejabberd#3572] [Questions] [answered]
I have the docker image installed on my pc and I have set up a Mnesia database
I am trying to see the contents of the database. Is there a function defined for this? How can I inspect ejabberd's Mnesia database?
I cant use tv:start() because is not define,

**Comment by mremond** [comment, marked as answer]
The Erlang `tv` application is not included in the binary installer or in the Docker image. You thus cannot start that TV application.

You can try using `ejabberdctl dump` command to dump the database into a text file to check its content.

**Comment by badlop** [comment]
> I am trying to see the contents of the database.

If you just want to check the tables sizes, and number of elements, you can browse ejabberd WebAdmin -> Nodes -> your node -> Database
